src/main/java/core/basesyntax/model/Transaction.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax.model;
+
+public class Transaction {
+    private final String operation;
+    private final String fruitName;
+    private final int quantity;
+
+    public Transaction(String operation, String fruitName, int quantity) {
+        this.operation = operation;
+        this.fruitName = fruitName;
+        this.quantity = quantity;
+    }
+
+    public String getOperation() {
+        return operation;
+    }
+
+    public String getFruitName() {
+        return fruitName;
+    }
+
+    public int getQuantity() {
+        return quantity;
+    }
```

Comment:
Ensure the fields of the Transaction class are adequately protected and validated. If there are specific constraints or rules for the operation, fruitName, or quantity, consider adding validation logic in the constructor or using setters with validation. This will help adhere to the encapsulation principle and maintain data integrity.